h5c/vic/src/vic-webapp/src/app/shared/components/vch-network/vch-network.component.html

```diff
@@ -0,0 +1,823 @@
+            </clr-stack-block>
+          </ng-template>
+          <clr-stack-block>
+            <clr-stack-label>DNS</clr-stack-label>
+            <clr-stack-content>{{ container.containerNetworkDns }}</clr-stack-content>
+          </clr-stack-block>
+          <clr-stack-block>
+            <clr-stack-label>{{ i18n.translate('vch.storage.network.label') }}</clr-stack-label>
+            <clr-stack-content>{{ container.containerNetworkLabel }}</clr-stack-content>
+          </clr-stack-block>
+          <clr-stack-block>
+            <clr-stack-label>{{ i18n.translate('vch.storage.network.firewallPolicy') }}</clr-stack-label>
+            <clr-stack-content>{{ container.containerNetworkFirewall }}</clr-stack-content>
+          </clr-stack-block>
+        </clr-stack-block>
+      </ng-container>
+
+    </clr-stack-view>
+  </ng-container>
+</div>
+
+<!-- Form content -->
+<ng-template #formContent>
+  <div class="spinner" *ngIf="portgroupsLoading"></div>
+  <div *ngIf="!portgroupsLoading">
+    <p class="mt-0"> {{ i18n.translate('vch.storage.network.specifyNetworksVCH') }}</p>
+    <a class="btn btn-link pl-0"
+       href="https://vmware.github.io/vic-product/assets/files/html/1.4/vic_vsphere_admin/vch_networking.html"
+       target="_blank">{{ i18n.translate('vch.storage.network.viewNetworksRequirements') }}<clr-icon shape="pop-out"></clr-icon>
+    </a>
+
+    <form class="pt-0" [formGroup]="form" novalidate>
+      <section class="form-block my-0"  id='networks'>
+
+        <!-- Required VCH Networks -->
+        <label>{{ i18n.translate('vch.storage.network.requiredVchNetworks') }}</label>
+
+        <!-- Bridge Network -->
+        <div class="row form-group mb-0">
+          <div class="col-xs-3">
+            <label class="required" for="bridge-network-selector">{{ i18n.translate('vch.storage.network.bridgeNetwork') }}</label>
+          </div>
+          <div class="col-xs">
+            <div class="select form-control">
+              <select id="bridge-network-selector" formControlName="bridgeNetwork">
+                <option value="">{{ i18n.translate('vch.storage.network.notConfigured') }}</option>
+                <option [value]="portgroup['text']" *ngFor="let portgroup of portgroups">{{ portgroup.text }}</option>
+              </select>
+            </div>
+          </div>
+          <div class="col-xs text-right">
+            <label for="bridge-network-range-selector">{{ i18n.translate('vch.storage.network.bridgeNetworkRange') }}</label>
+          </div>
+          <div class="col-xs">
+            <label for="bridge-network-range-selector"
+                   aria-haspopup="true"
+                   role="tooltip"
+                   class="tooltip tooltip-validation tooltip-lg tooltip-top-left"
+                   [class.invalid]="form.get('bridgeNetworkRange').invalid && (form.get('bridgeNetworkRange').dirty || form.get('bridgeNetworkRange').touched)">
```

Comment:
I thought you had written a util function to check for isInvalid? If so, it seems could be used here and other places in this template instead of inline.